### PR TITLE
Documentation for deployment and services 

### PR DESCRIPTION
**Pull request by megandouli**
# what
Additional docs for deployment

### CODE REVIEW

File: documentation/services.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+CCA uses the folling 3rd party services. The site depends on most of these being up to run correctly. Some are just used for analytics.
+
+## Notifications
+
+- Twilio
+
+## Deployment
+
+- Terraform and AWS for deployment
+- Cloudflare for Domain Name System (DNS) management
```
Comment (bugsalexander) [inline documentation/services.md:12]: i think dns is managed by terraform also–just learned this today from @gamburgm , actually! https://github.com/sandboxnu/dns

Comment (megandouli) [reply documentation/services.md:12]: I'm pretty sure it's Cloudflare for SearchNEU... we're on Ryan Hughes's account still

Comment (bugsalexander) [reply documentation/services.md:12]: oh yes you are so right my bad sorry

File: documentation/services.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Services
+
+CCA uses the folling 3rd party services. The site depends on most of these being up to run correctly. Some are just used for analytics.
```
Comment (bugsalexander) [inline documentation/services.md:3]:
```suggestion
CCA uses the following 3rd party services. The site depends on most of these being up to run correctly. Some are just used for analytics.
```

**Pull request merged**